index.html

```diff
@@ -4292,6 +4306,493 @@ <h3>Proof Formats</h3>
+          </li>
+          <li>
+zero or more warnings (array of <a>ProblemDetails</a> <var>warnings</var>)
+          </li>
+          <li>
+zero or more errors (array of <a>ProblemDetails</a> <var>errors</var>)
+          </li>
+        </ul>
+
+        <p>
+The verification algorithm is as follows:
+        </p>
+
+        <ol class="algorithm">
+          <li>
+Set <var>verified</var> to an empty object.
+          </li>
+          <li>
+Ensure that the securing mechanism has properly protected the
+<var>conforming document</var> by performing the following steps:
+            <ol class="algorithm">
+              <li>
+Set <var>securingMechanismVerified</var> to the result of executing the
+<a href="#verify-securing-mechanism">securing mechanism verification
+algorithm</a> with <var>inputBytes</var> and <var>mediaType</var> passed as
+inputs to the algorithm.
+              </li>
+              <li>
+Set <var>status</var>, <var>document</var>, <var>mediaType</var>,
+<var>warnings</var>, and <var>errors</var> in <var>verified</var> to their
+respective properties in <var>securingMechanismVerified</var>.
+              </li>
+              <li>
+If <var>verified</var>.<var>status</var> is `false`, the provided
+<var>inputBytes</var> contains a document that cannot be considered to be
+secured. Return <var>verified</var>, which will contain warning and error
+information related to the verification failure.
+              </li>
+            </ol>
+          </li>
+          <li>
+Ensure that the <var>verified</var>.<var>document</var> is a
+<a>conforming document</a> by performing the following steps:
+            <ol class="algorithm">
+              <li>
+Set <var>dataModelVerified</var> to the result of executing the
+<a href="#verify-data-model">data model verification algorithm</a>
+with <var>document</var> passed as input to the algorithm.
+              </li>
+              <li>
+Set <var>status</var>, <var>document</var>, <var>warnings</var>, and
+<var>errors</var> in <var>verified</var> to their respective properties in
+<var>dataModelVerified</var>.
+              </li>
+              <li>
+If <var>verified</var>.<var>status</var> is `false`, the provided
+<var>inputBytes</var> contains a non-conforming document. Return
+<var>verified</var>, which will contain warning and error information
+related to the verification failure.
+              </li>
+            </ol>
+          </li>
+          <li>
+Return <var>verified</var>.
+          </li>
+        </ol>
+
+        <p>
+The steps for verifying the state of the securing mechanism and verifying
+that the input document is a <a>conforming document</a> MAY be performed in
+a different order than the one provided above as long as the
+implementation returns errors for the same inputs.
+Implementations MAY produce different errors than described above.
+        </p>
+
+      </section>
+
+      <section class="normative">
+        <h3>Verify Data Model</h3>
```

Comment:
```suggestion
        <h3>Validate Data Model</h3>
```

Reply:
Fixed in 5b02e7b53d078d41ab2323ce6a27cab0b8ea6dd3.

Reply:
Thanks, Resolving.